### ISSUE TITLE
[Idea] Motion Blur for GIF/Animation Export

### ISSUE THREAD

**Issue by mattdesl** [mattdesl/canvas-sketch#10]
Just saw [this](https://necessarydisorder.wordpress.com/2018/07/02/getting-started-with-making-processing-gifs-and-using-the-beesandbombs-template/) post where the author writes about a motion blur technique for processing GIF loops.

https://gist.github.com/Bleuje/e06d3b23abe45c4d04973b8b0f90b7a4

I think something like that could be interesting to explore; maybe built-in, or as a utility, or just as an example.

**Comment by guidoschmidt** [comment]
Motion blur would be awesome - I guess starting with just an example would be a good starting point 🤔

**Comment by fturcheti** [comment]
I agree that an example seems a better approach for now. And when the "External Module for utilities (randomness, geometry, etc)" takes form, it can be served as an utility.

**Comment by delucis** [comment]
I just implemented this in my Node sketch renderer:

https://github.com/delucis/pellicola/blob/735bd7487bdc597ac7272e4ddce9473c15f68d09/lib/frame-maker.js#L99-L134

Don’t have time right now to look into how it could be folded into `canvas-sketch`, but perhaps the work there might be useful.

**Comment by mattdesl** [comment]
Thanks @delucis! I made a small example here as a proof of concept. It's not bullet-proof but I may use something like this in my future sketches. :smile:

https://gist.github.com/mattdesl/447feabf4c819889a5e73de0da37abc0

<img src="https://user-images.githubusercontent.com/1383811/48987350-f44b8f80-f115-11e8-918a-d1f3c25c95c9.gif" width="50%" />
